request_handler.py

```diff
@@ -0,0 +1,75 @@
+import importlib
+import methods
+import copy
+
+def execute(request):
+    """
+    Executing requests using the methods defined in the methods module.
+    
+    Request dict format:
+    {"func" : <function name that is in the methods module>,
+     "requests" : <list of dicts, functions and arguments to run in order. Not used if "func" is not equal to "tether">, 
+     "args" : <ordered arguments for func, including data input. For tether, these are the initial input arguments>,
+     "kwargs" : <key worded arguments for func>
+    }
+    
+    input: request (dict)
+    output: <tether function> OR method.func(*args, **kwargs)
+    """
+    
+    args = []
+    kwargs = {}
+    
+    if "args" in request:
```

Comment:
why not:
```
args = []
if "args" in request and request["args"] != None:
    args = request["args"]
```

Reply:
```
if request.get('args', None)  is not None:
    args = request['args']
```

Reply:
Valid python is probably better, I can only offer pseudo-Python. :grin: :snake: